Lib/fontTools/cffLib/specializer.py

```diff
@@ -62,12 +63,58 @@ def programToCommands(program):
 		commands.append(('', stack))
 	return commands
 
-def commandsToProgram(commands):
+
+def commandsToProgram(commands, var_model=None, round_func=None):
 	"""Takes a commands list as returned by programToCommands() and converts
-	it back to a T2CharString program list."""
+	it back to a T2CharString or CFF2Charstring program list."""
 	program = []
-	for op,args in commands:
-		program.extend(args)
+	for op, args in commands:
```

Comment:
Can you please explain what this piece of code does?  Probably should be added as comments.

Reply:
Right.  Agree with both points.